h2o-extensions/xgboost/src/main/java/hex/tree/xgboost/XGBoostModel.java

```diff
@@ -440,7 +446,24 @@ protected XGBoostModel deepClone(Key<XGBoostModel> result) {
   }
   
   private static int getMaxNThread() {
-    return Integer.getInteger(SYSTEM_PROP_PREFIX + "xgboost.nthread", H2O.ARGS.nthreads);
+    if (System.getProperty(PROP_NTHREAD) != null) {
```

Comment:
What will happen when there are other H2O clusters running on the same physical server? Would that be an issue?

Reply:
yes, that can cause issues